src/modules/aws_vpc/entity/route.ts

```diff
@@ -23,124 +24,123 @@ export class Route {
    * Reference to the route table associated with this route
    */
   @ManyToOne(() => RouteTable, {
+    eager: true,
+    nullable: false,
     onDelete: 'CASCADE',
-    onUpdate: 'CASCADE',
   })
-  @JoinColumn()
+  @JoinColumn([
+    {
+      name: 'route_table_id',
+      referencedColumnName: 'id',
+    },
+    {
+      name: 'region',
+      referencedColumnName: 'region',
+    },
+  ])
   routeTable: RouteTable;
 
   /**
    * @public
-   * destination fields: used to determine the destination to be matched
+   * destination field: used to determine the destination to be matched. Could be an IPv4 or IPv6 CIDR block or a prefixed list
    */
-  @Column({ nullable: true })
-  @cloudId
-  DestinationCidrBlock?: string;
-
-  /**
-   * @public
-   * destination fields: used to determine the destination to be matched (ipv6)
-   */
-  @Column({ nullable: true })
-  @cloudId
-  DestinationIpv6CidrBlock?: string;
-
-  /**
-   * @public
-   * A managed prefix list is a set of one or more CIDR blocks.
-   * @see https://docs.aws.amazon.com/vpc/latest/userguide/managed-prefix-lists.html
-   */
-  @Column({ nullable: true })
-  @cloudId
-  DestinationPrefixListId?: string;
+  @Check(`
+    "destination" ~ '^(?:(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)\.){3}(?:25[0-5]|2[0-4][0-9]|[01]?[0-9][0-9]?)/(3[0-2]|[12][0-9]|[0-9])$'
+    OR "destination" LIKE 'pl-%'
```

Comment:
I was thinking of this one as the *first* in the list since it's the fastest to check.